tools/block-generator/README.md

```diff
@@ -10,19 +10,20 @@ Block generator uses a YAML config file to describe the composition of each rand
 2. Transaction type distribution
 3. Transaction type specific configuration
 
-At the time of writing, the block generator supports **payment** and **asset** transactions. The settings are hopefully, more or less, obvious. Distributions are specified as fractions of 1.0, and the sum of all options must add up to 1.0.
+The block generator supports **payment**, **asset**, and **application** transactions. The settings are hopefully, more or less, obvious. Distributions are specified as fractions of 1.0, and the sum of all options must add up to ~1.0.
```

Comment:
I think we should stay with `all options must add up to 1.0`.  It's less confusing.